addon/components/ui-popup.js

```diff
@@ -3,9 +3,9 @@ import Base from '../mixins/base';
 
 export default Ember.Component.extend(Base, {
   module: 'popup',
-  attributeBindings: ['content:data-content'],
-  
-  contentChanges: Ember.observer('content', function() {
-    this.didInsertElement();
-  })
+
+  didUpdate() {
+    this._super(...arguments);
+    this.initializeModule();
```

Comment:
We only need to reinitialize the popup, we can skip the rest of the did insert bindings